packaging/neo4j-desktop/src/main/java/org/neo4j/desktop/runtime/DesktopConfigurator.java

```diff
@@ -55,9 +54,9 @@ public DesktopConfigurator( Installation installation, Parameters parameters, Fi
     public void refresh()
     {
         config = new ConfigLoader( CommunityBootstrapper.settingsClasses).loadConfig(
+                dbDir.getAbsoluteFile(),
                 Optional.of( getConfigurationsFile() ),
                 FormattedLog.toOutputStream( System.out ),
-                (settings) -> settings.put( GraphDatabaseSettings.neo4j_home.name(), dbDir.getAbsolutePath() ),
```

Comment:
I think you can get rid of the "customizer" mechanism now that this has gone. It was only introduced for this purpose.